### PR TITLE
overall review

### PR DESCRIPTION
**Pull request by tomkralidis**

### CODE REVIEW

File: workshop/content/docs/introduction.md [hunk 1]
```diff
@@ -44,9 +45,16 @@ The above examples may help as starting points for your own pygeoapi setup and d
...
-components such as GDAL, rasterio, Shapely, Pandas, Elasticsearch, PostGIS and many others.
+code sprints were instrumental for agile development and pouring the foundation of the project.
+
+The core design principles are as follows:
+
+- simplicity / low barrier to entry
+- long term sustainability
+- modularity
+- extensibility
+- building by exception based on a large ecosystem of Free Open Source and OSGeo
```
Comment (justb4) [inline workshop/content/docs/introduction.md:56]: *"..building by exception based on a large ecosystem of Free Open Source and OSGeo..."*
Maybe I don't understand what "building by exception" means, but would simply phrase: 
*"..building on a large ecosystem of Free and Open Source and OSGeo..."*

File: workshop/content/docs/publishing/first.md [hunk 2]
```diff
@@ -13,18 +13,18 @@ You can find this dataset in `workshop/exercises/data/free-wifi-florence.csv`.
 
 This exercise consists of two key steps:
 
-* adapt the `pygeoapi.config.yml` to define this dataset as an OGC API - Features *collection*
-* ensure that pygeoapi can find and connec to the data file
+* adapt the `workshop/exercises/pygeoapi.config.yml` to define this dataset as an OGC API - Features *collection*
```
Comment (justb4) [inline workshop/content/docs/publishing/first.md:16]: suggest:

* adapt `workshop/exercises/pygeoapi.config.yml` to define this dataset as an OGC API - Features *Collection*

**Review by justb4 — CHANGES_REQUESTED** [review]
All, some small phrasing suggestions, but may leave as well.

**Comment by justb4** [comment]
Missed one in [index.md L18](https://github.com/geopython/diving-into-pygeoapi/blob/main/workshop/content/docs/index.md?plain=1#L18)

# Your [FOSS4G 2022](https://2022.foss4g.org) workshop team
to
# Your [FOSS4G 2023](https://2023.foss4g.org) workshop team

**Comment by tomkralidis** [comment]
thanks @justb4.  Updated for review.

**Comment by pvgenuchten** [comment]
Looks good, thanx Tom

**Pull request merged**